tox.ini

```diff
@@ -4,7 +4,7 @@
 # directory.
 
 [tox]
-envlist = py{27,py27,36,37,38,39,py36,py37}
+envlist = py{37,38,39,py36,py37}
```

Comment:
Guess py36 can be removed as well or better replaced by py38 and py39?

Reply:
nice catch, done.

Reply:
please approve if you have no further comments.